compiler/src/dotty/tools/dotc/core/tasty/TastyPrinter.scala

```diff
@@ -258,10 +258,10 @@ class TastyPrinter(bytes: Array[Byte]) {
       import reader.*
       sb.append(s" ${reader.endAddr.index - reader.currentAddr.index}")
```

Comment:
this should be removed - currently it prints something like ` 2Attributes (2 bytes, starting from 602):`
```suggestion
```